packages/client/hmi-client/src/components/Header.vue

```diff
@@ -15,7 +16,9 @@ const goToHomepage = () => {
 };
 const goToDataExplorer = () => emit('show-data-explorer');
 
-const projectName = 'Name of the project that can be long for clarity and precision';
+defineProps<{
+	projectName?: Project['name'];
```

Comment:
Depending on your response to my other comment you may be able to take this `?` out as well.
```suggestion
	projectName: Project['name'];
```